index.html

```diff
@@ -5612,25 +5533,174 @@ <h3>Remote Document and Context Retrieval</h3>
+              having an <a data-cite="HTML/dom.html#the-id-attribute">id attribute</a>
+              that matches the fragment identifier, after decoding <a data-cite="RFC3986#section-2.1">percent encoded sequences</a>.
+              <p>If no such element is found,
+                or the located element is not a <a>JSON-LD script element</a>,
+                the <var>promise</var> is rejected with a <a>JsonLdError</a> whose code is set to <a data-link-for="JsonLdErrorCode">loading document failed</a>
+                and processing is terminated.</p>
+            </li>
+            <li>Otherwise, if the <a data-link-for="LoadDocumentOptions">profile</a>
+              option is specified,
+              set <var>source</var> to the result of transforming the
+              <a data-cite="DOM#dom-node-textcontent">textContent</a>
+              of the first <a data-cite="HTML/scripting.html#the-script-element">script element</a> in <var>document</var>
+              having an <a data-cite="HTML/semantics.html#attr-link-type">type attribute</a>
+              of <code>application/ld+json</code> along with the value of the
+              <a data-link-for="LoadDocumentOptions">profile</a> option, if found.</li>
+            <li>If <var>source</var> is undefined and the <a data-link-for="LoadDocumentOptions">extractAllScripts</a> option is not present, or <code>false</code>,
+              set <var>source</var> to the <a data-cite="DOM#dom-node-textcontent">textContent</a>
+              of the first <a>JSON-LD script element</a> in <var>document</var>.
+              <p>If no such element is found,
+                or the located element is not a <a>JSON-LD script element</a>,
+                the <var>promise</var> is rejected with a <a>JsonLdError</a> whose code is set to <a data-link-for="JsonLdErrorCode">loading document failed</a>
+                and processing is terminated.</p></li>
+            <li>If <var>source</var> is defined and the,
+              set <var>document</var> to the result of the
+              <a href="#extract-script-content">Extract Script Content algorithm</a>,
+              using <var>source</var>, rejecting <var>promise</var>
+              with a <a>JsonLdError</a> whose code set from the result, if an error is detected
+              and processing is terminated.
+            </li>
+            <li>Otherwise, if <var>source</var> is undefined and the <a data-link-for="LoadDocumentOptions">extractAllScripts</a> option is not present, or <code>false</code>,
+              the <var>promise</var> is rejected with a <a>JsonLdError</a> whose code is set to <a data-link-for="JsonLdErrorCode">loading document failed</a>
+              and processing is terminated.</li>              
+            <li>Otherwise, if <var>source</var> is undefined and the <a data-link-for="LoadDocumentOptions">extractAllScripts</a> option is <code>true</code>,
```

Comment:
If I understand correctly, this "otherwise" is an "else", but it reads like an "else if", which might confuse the reader.
I suggest rephtrasing it (1/2)
```suggestion
            <li>Otherwise,<var>source</var> is undefined and the <a data-link-for="LoadDocumentOptions">extractAllScripts</a> option is <code>true</code>.
```

Reply:
I added some deeper lists to clarify.